meteor/server/api/blueprints/context/context.ts

```diff
@@ -59,17 +59,28 @@ import { ExternalMessageQueue } from '../../../../lib/collections/ExternalMessag
 import { extendIngestRundownCore } from '../../ingest/lib'
 import { loadStudioBlueprint, loadShowStyleBlueprint } from '../cache'
 import { CacheForRundownPlaylist, ReadOnlyCacheForRundownPlaylist } from '../../../DatabaseCaches'
-import { getSelectedPartInstancesFromCache } from '../../playout/lib'
+import { BlueprintId } from '../../../../lib/collections/Blueprints'
+
+export interface ContextInfo {
+	name: string
```

Comment:
This could benefit from some documentation. My first question is: what's the difference between "name" and "identifier"?